src/style.css

```diff
@@ -0,0 +1,106 @@
+/* Sometimes test on BackstopJS passes with 3 passed and 3 failed.
+   Sometimes it's 5 passed and 1 failed. So I just send it. */
+
+@font-face {
+  font-family: Avenir;
+  src: url(./Avenir.ttc);
+  font-weight: 300;
+}
+
+.searchbar {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-bottom: 20px;
+}
+
+.searchbar-large {
```

Comment:
i think you dont need this class, just make .searchbar as default and apply to it default sizes, and then make .searchbar-small and override necessary styles